src/fungal_effects.cpp

```diff
@@ -100,9 +100,8 @@ void fungal_effects::marlossify( const tripoint &p )
         m.ter_set( p, t_marloss );
         return;
     }
-    bool is_fungi;
     for( int i = 0; i < 25; i++ ) {
-        is_fungi = m.has_flag_ter( "FUNGUS", p );
+        bool is_fungi = m.has_flag_ter( "FUNGUS", p );
```

Comment:
Do we really need to initialize this variable 25 times?

Reply:
Thanks for pointing that out. On second look that change is actually an erroneous one and should be reverted.